src/ui/server/controllers/insurance/export-contract/agent-details/index.ts

```diff
@@ -157,7 +157,8 @@ export const post = async (req: Request, res: Response) => {
       return res.redirect(PROBLEM_WITH_SERVICE);
     }
 
-    return res.redirect(`${INSURANCE_ROOT}/${referenceNumber}${AGENT_SERVICE}`);
+    // return res.redirect(`${INSURANCE_ROOT}/${referenceNumber}${AGENT_SERVICE}`);
```

Comment:
remove comment?

Reply:
Could do, although it's only going to be replaced with the commented line in the next PR for `AGENT_SERVICE`